terraform/modules/external_domain_broker_tests/resources.tf

```diff
@@ -19,12 +19,105 @@ resource "aws_route53_record" "record" {
     "${aws_route53_zone.zone.name_servers.3}",
   ]
 }
+
+resource "aws_route53_zone" "stack_test_0" {
+  name    = "${var.stack_description}.cloud-gov-test-domain-0.com"
+  comment = "Hosts TXT and CNAME records for the external-domain-broker tests"
+}
+
+data "aws_route53_zone" "test_0" {
+  name = "cloud-gov-test-domain-0.com"
+}
+resource "aws_route53_record" "record" {
+  name    = "${aws_route53_zone.stack_test_0.name}"
+  zone_id = "${data.aws_route53_zone.test_0.zone_id}"
+  type    = "NS"
+  ttl     = "60"
+
+  records = [
+    "${aws_route53_zone.test_0.name_servers.0}",
```

Comment:
Is this pointing to line 23?

Reply:
:doh:

Reply:
It is pointing to line 28, no?

Reply:
yeah, it's pointing to 28 but it's supposed to be pointing to 23
updated in d04bcdf10c30de926cd60969501b810072fe4ece